packages/gatsby-plugin-google-tagmanager/README.md

```diff
@@ -24,6 +24,11 @@ plugins: [
       gtmAuth: "YOUR_GOOGLE_TAGMANAGER_ENVIROMENT_AUTH_STRING",
       gtmPreview: "YOUR_GOOGLE_TAGMANAGER_ENVIROMENT_PREVIEW_NAME",
       dataLayerName: "YOUR_DATA_LAYER_NAME",
+
+      // Whether to put the GTM script into the <head> (as suggested by Google)
+      // or append it to the <body> (making it non-blocking).
+      // If you activate Google Optimize through Google Tag Manager leave it untouched.
+      placeInHead: true,
```

Comment:
Since the current default is to place it in HEAD, I would flip this over and add an option to explicitly place this in body (`addTagInBody` or whatever) so that we don't change default behaviour

Reply:
It makes sense! I've just pushed the change 😊
Thank you @sidharthachatterjee